lms/urls.py

```diff
@@ -97,6 +97,11 @@
     url(r'', include('openedx.features.specializations.urls')),
     url(r'^marketplace/', include('openedx.features.marketplace.urls')),
     url(r'^idea/', include('openedx.features.idea.urls')),
+    url(r'^organizations/', TemplateView.as_view(template_name='features/experiments/organizations.html')),
```

Comment:
Let's please add a comment here that these are temporary experiment urls.

Reply:
@qasimgulzar why are we using 5 different templates since we need to keep them empty? @ahmed-faraz46 @mabdullahz @zeshanarifios thoughts?

Reply:
update

Reply:
Adding to imran bhai's point why don't we use "main_django" directly here?